src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -152,4 +168,27 @@ private static boolean isSwitchExpression(DetailAST ast) {
         return ast.getParent().getType() == TokenTypes.EXPR
                 || ast.getParent().getParent().getType() == TokenTypes.EXPR;
     }
+
+    /**
+     * Checks if the case contains null label.
+     *
+     * @param ast the switch statement we are checking
+     * @return returnValue the ast of null label
+     */
+    private static DetailAST verifyNullCaseLabel(DetailAST ast) {
+        DetailAST returnValue = null;
+        for (DetailAST ast1 = ast; ast1 != null; ast1 = ast1.getNextSibling()) {
```

Comment:
We have no need for any iteration here, we can and should be be much more deliberate about what we are checking.

Reply:
Removed Iteration @nrmancuso